pipeline/load.py

```diff
@@ -0,0 +1,288 @@
+"""This is the script to load plant data into the database"""
+from os import environ
+from datetime import datetime
+import logging
+import ast
+import pandas as pd
+import psycopg2
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def config_log() -> None:
+    """
+    Terminal logs configuration.
+    """
+    logging.basicConfig(
+        format="{asctime} - {levelname} - {message}",
+        style="{",
+        datefmt="%Y-%m-%d %H:%M",
+        level=logging.INFO,
+    )
+
+
+def get_connection() -> psycopg2.extensions.connection:
+    """
+    Tries to connect to the RDS database.
+    Returns a connection if successful.
+    """
+
+    logging.info("Connecting to the database.")
+
+    try:
+        connection = psycopg2.connect(
+            host=environ["DB_HOST"],
+            port=environ["DB_PORT"],
+            user=environ["DB_USER"],
+            password=environ["DB_PASSWORD"],
+            database=environ["DB_NAME"]
+        )
+        return connection
+    except psycopg2.OperationalError:
+        logging.warning("The database %s doesn't exist", environ["DB_NAME"])
+        return None
+
+
+def get_cursor(connection: psycopg2.extensions.connection) -> psycopg2.extensions.cursor:
+    """
```

Comment:
this is just a tiny thing, but the function potentially can return `None` so could make the type hint:
`def get_cursor(connection: psycopg2.extensions.connection) -> Optional[psycopg2.extensions.cursor]:`
to make this clear